test/e2e-tests/README.md

```diff
@@ -36,10 +37,12 @@ The following enviroment variables are required to run the tests. If any of thes
 
 The following enviroment variables are optional. You may still need to set them for the tests to pass depending on your setup. Read [here](../../pipelines/README.md#ai-edge-end-to-end-pipeline) for more context on how to set these up.
 
-- `SELF_SIGNED_CERT` - A path on your local machine to a self signed cert to be used in pipeline
+- `GIT_SELF_SIGNED_CERT` - Self-signed cert to be used by git when cloning
+- `S3_SELF_SIGNED_CERT` - Self-signed cert to be used for S3 when pulling models
 - `GO` - Custom Go installation path that is not set in your `PATH`
 
 ```bash
+- `S3_SELF_SIGNED_CERT` - Self signed cert to be used for S3 when pulling models
```

Comment:
```suggestion
```

^ this looks like a mistake - it's already listed on line 41, and doesn't seem to make sense here inside this code block

Reply:
Created a small follow up https://github.com/opendatahub-io/ai-edge/pull/248